### PR TITLE
Request from players

### PR DESCRIPTION
**Pull request by brostosjoined**

### CODE REVIEW

File: plugins/minigames.json [hunk 1]
```diff
@@ -967,6 +967,139 @@
           "md5sum": "1cbe5b3e85b5dfcee1eb322f33568fd4"
         }
       }
+    },
+    "Avalanche": {
```
Comment (rikkolovescats) [inline plugins/minigames.json:971]: let's try stick to snake case plugin names here and for rest of the plugins in this PR

Comment (brostosjoined) [reply plugins/minigames.json:971]: AAh ok

File: plugins/minigames.json [hunk 2]
```diff
@@ -967,6 +967,424 @@
...
+        "1.0.0": {
+          "api_version": 8,
+          "commit_sha": "4941d0c",
+          "released_on": "01-02-2024",
+          "md5sum": "daf6c3e3d8663fa6d99c9e3f75033f36"
+        }
+      }
+    },
+    "fat_pigs": {
+      "description": "Survive...",
```
Comment (rikkolovescats) [inline plugins/minigames.json:1352]: not very descriptive

File: plugins/minigames.json [hunk 3]
```diff
@@ -967,6 +967,424 @@
...
+        "1.0.0": {
+          "api_version": 8,
+          "commit_sha": "4941d0c",
+          "released_on": "01-02-2024",
+          "md5sum": "fe0d82c322ce01d8956af3131e135ad2"
+        }
+      }
+    },
+    "flag_day": {
+      "description": "Pick up flags to receive a prize.\nBut beware...",
```
Comment (rikkolovescats) [inline plugins/minigames.json:1371]: we shouldn't need `\n` line breaks in descriptions, plugman should automatically do that for us

File: plugins/minigames.json [hunk 4]
```diff
@@ -967,6 +967,424 @@
...
+        "1.0.0": {
+          "api_version": 8,
+          "commit_sha": "4941d0c",
+          "released_on": "01-02-2024",
+          "md5sum": "1259e09ccd9b10ae82e4a6623572a4d2"
+        }
+      }
+    },
+    "explodo_run": {
+      "description": "Run For Your Life :))",
```
Comment (rikkolovescats) [inline plugins/minigames.json:1314]: run for my life from whom? :upside_down_face:

File: plugins/minigames.json (hunk 2 above)
Comment (rikkolovescats) [inline plugins/minigames.json:1352]: let's make this a lil more descripty

**Comment by rikkolovescats** [comment]
some plugins are missing names for authors, best if we can find out who made those, otherwise let's add `(Unknown)` or something else for name.

i'll review it in more depth, maybe on weekend.

**Comment by rikkolovescats** [comment]
some plugins are missing names for authors, best if we can find out who made those, otherwise let's add `(Unknown)` or something else for name.

i'll try review it in more depth, maybe on weekend.

**Comment by brostosjoined** [comment]
Okay will check for the names and stuff and probably more fixes if possible

**Comment by brostosjoined** [comment]
pull

**Review by rikkolovescats — DISMISSED** [review]
thanks so much here @brostosjoined!

**Comment by Loup-Garou911XD** [comment]
Big w

**Pull request merged**